src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -34,7 +34,7 @@ SegmentEncodingSpec get_segment_encoding_spec(const BaseEncodedSegment& base_enc
         return {base_encoded_segment.encoding_type(), VectorCompressionType::SimdBp128};
     }
 
-    Fail("GCC thinks this is reachable");
+    Fail("Invalid enum value");
```

Comment:
Fun fact: GCC is right. https://stackoverflow.com/questions/33607284/control-reaches-end-of-non-void-function-with-fully-handled-case-switch-over-a